scipy/stats/morestats.py

```diff
@@ -1098,22 +1098,23 @@ def boxcox_normmax(x, brack=None, method='pearsonr', bounds=None,
     optimize_method : str or callable, optional
         Type of solver.  Should be one of:
 
-            - 'Brent'     :ref:`(see here) <optimize.minimize_scalar-brent>`
-            - 'Bounded'   :ref:`(see here) <optimize.minimize_scalar-bounded>`
-            - 'Golden'    :ref:`(see here) <optimize.minimize_scalar-golden>`
-            - custom - a callable object
+        - 'Brent'     :ref:`(see here) <optimize.minimize_scalar-brent>`
+        - 'Bounded'   :ref:`(see here) <optimize.minimize_scalar-bounded>`
+        - 'Golden'    :ref:`(see here) <optimize.minimize_scalar-golden>`
+        - custom - a callable object
+
     tol : float, optional
         Tolerance for termination. For detailed control, use solver-specific
         options. See `scipy.optimize.minimize_scalar` for more details.
     options : dict, optional
         A dictionary of solver options.
 
-            maxiter : int
-                Maximum number of iterations to perform.
-            disp : bool
-                Set to True to print convergence messages.
+        maxiter : int
+            Maximum number of iterations to perform.
+        disp : bool
+            Set to True to print convergence messages.
 
-        See `scipy.optimize.minimize_scalar` for more details.
+        See :func:`minimize_scalar` for more details.
```

Comment:
Was this an accident? I had fixed this and a few others to take care of the CircleCI error:
```
/home/circleci/repo/build/testenv/lib/python3.7/site-packages/scipy/stats/morestats.py:docstring of scipy.stats.boxcox_normmax:59: WARNING: py:func reference target not found: minimize_scalar

```

Reply:
Oops yes looks like it.

Reply:
Ok, well this is causing one of the failures.